<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lCurrentTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущее время: 0:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAlarms">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>158</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить
будильники</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTimers">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>158</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить
таймеры</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="saTimers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="sawcTimers">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>449</width>
      <height>259</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbAddTimer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить таймер</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddAlarm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить будильник</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
